e2e/test_bootstrap_parallel_multiple_versions.sh

```
@@ -0,0 +1,114 @@
+# subdirectory to avoid concurrent writes to the shared staging directory.
+# "process-install-deps" and "complete" run in parallel for the versions that
+# successfully build (flit-core >=3.10.0, which are compatible with Python 3.14+).
+PARALLEL_PHASES="start prepare-source prepare-build build process-install-deps complete"
+PHASES_WITHOUT_PARALLEL_BATCH=""
+
+echo ""
+echo "Checking that parallel phases ran with at least one multi-item batch..."
+for phase in $PARALLEL_PHASES; do
+  max_items=0
+  while IFS= read -r line; do
+    count=$(echo "$line" | sed 's/.*items=\([0-9]*\).*/\1/')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)